crates/engine/local/src/payload.rs

```diff
@@ -59,3 +59,19 @@ where
         }
     }
 }
+
+/// A temporary workaround to support local payload engine launcher for arbitrary payload
+/// attributes.
+// TODO(mattsse): This should be reworked so that LocalPayloadAttributesBuilder can be implemented
+// for any
```

Comment:
This would be super helpful - have gotten stuck here a few times in the past

Reply:
ack, tracking this now